tests/dbcsr_unittest1.F

```diff
@@ -320,6 +333,8 @@ PROGRAM dbcsr_unittest
    ! clean mp environment
    CALL dbcsr_mp_release(mp_env)
 
+   call dbcsr_mp_grid_remove(mp_env)
```

Comment:
why is this needed? `mp_release` seems to `grid_remove` internally? Furthermore, the `refcount` of `mp_env` should reach 0 inside the `mp_release` (since we're at the end), meaning that `mp_env%mp` should already be deallocated

Reply:
No, we discussed it with Alfio already. It should be here otherwise it reports about comm leaks. Also, it cannot be built in dbcsr_mp_release, because some of the tests fail, probably because of some manipulations in Cannon's files. Maybe it should be put before `dbcsr_mp_release`. But I dont remember.